### ISSUE TITLE
[Feature Request] pip audio background button

### ISSUE THREAD

**Issue by birdup000** [futo-org/grayjay-android#181]

**Comment by greengeckowizard** [comment]
The audio background feature where you hit it on the pip (picture in picture)

![Screenshot_20231105_011417](https://github.com/futo-org/grayjay-android/assets/34012548/84eab9d9-8f1c-4208-8228-f4f0ed2d4687)

[Image cropped from](https://www.howtogeek.com/324401/how-to-use-or-disable-picture-in-picture-mode-in-android-oreo/)

**Comment by nastynick64** [comment]
This has been implemented (If I understood what you wanted correctly)

![Image](https://github.com/user-attachments/assets/32507f24-ba33-412e-8945-565b531dce38)

**Comment by Zvonimir-FUTO** [comment]
Yes, this was implemented. Forgot to close the issue, thanks for bump.
